packages/code-studio/src/dashboard/linker/LinkerLink.tsx

```diff
@@ -15,28 +14,52 @@ const TRIANGLE_HYPOTENUSE = Math.sqrt(
 const TRIANGLE_THETA = Math.asin((TRIANGLE_BASE * 0.5) / TRIANGLE_HEIGHT);
 const CLIP_RADIUS = 15;
 
-export class LinkerLink extends PureComponent {
-  static makeCirclePath(x, y, r) {
+export type LinkerLinkProps = {
+  x1: number;
+  y1: number;
+  x2: number;
+  y2: number;
+  id: string;
+  className: string;
+  onClick: (id: string) => void;
```

Comment:
Since the param names in function types are just for documentation, you might be able to make the param name more descriptive? `clickedId` or `thisId` or `linkId`? Or leave it. Doesn't really matter

Reply:
I'm going to leave it